Dockerfile

```diff
@@ -84,3 +84,43 @@ ENV PORT=3000
 # https://nextjs.org/docs/app/api-reference/config/next-config-js/output
 ENV HOSTNAME="0.0.0.0"
 CMD ["node", "apps/melding-form/server.js"]
+
+
+################################################
+#                 BACK-OFFICE                  #
+################################################
+# Sourced from https://github.com/vercel/next.js/blob/canary/examples/with-docker/Dockerfile
+FROM base AS back_office
```

Comment:
A little nitpicky, but: the other apps are called `admin_meldingen` and `melding_form_meldingen`. Shouldn't this be `back_office_meldingen`?